package.json

```diff
@@ -10,7 +10,7 @@
     "@types/blessed": "^0.1.17",
     "babel-jest": "^27.0.6",
     "blessed": "^0.1.81",
-    "chalk": "^5.2.0",
+    "chalk": "^4.1.2",
```

Comment:
Open question: 
Can't you upgrade typescript instead? It's better to start without technical debt

Reply:
Upgrading to chalk v5 seems a bit cumbersome, with very little added value. I recommend keeping v4 for now. However you could upgrade to the latest TS version indeed.

Reply:
Done